src/app/zap-templates/zcl/data-model/chip/smoke-co-alarm-cluster.xml

```diff
@@ -14,73 +14,62 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<configurator>
-  <domain name="CHIP" />
-
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/app_clusters/SmokeCOAlarm.adoc
+Parameters: in-progress 
+Git: 
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
+  <domain name="CHIP"/>
   <cluster>
     <domain>Measurement &amp; Sensing</domain>
     <name>Smoke CO Alarm</name>
     <code>0x005C</code>
     <define>SMOKE_CO_ALARM_CLUSTER</define>
     <description>This cluster provides an interface for observing and managing the state of smoke and CO alarms.</description>
-
     <!-- Abbreviations used in descriptions -->
-    <tag name="SMOKE" description="Smoke Alarm" />
-    <tag name="CO" description="CO Alarm" />
-
+    <tag name="SMOKE" description="Smoke Alarm"/>
+    <tag name="CO" description="CO Alarm"/>
     <client tick="false" init="false">true</client>
     <server tick="false" init="false">true</server>
-
     <!-- Current cluster version -->
-    <globalAttribute side="either" code="0xFFFD" value="1" />
+    <globalAttribute side="either" code="0xFFFD" value="1"/>
     <!-- Cluster feature map -->
-    <globalAttribute side="server" code="0xFFFC" type="Feature" value="3" />
-
     <features>
       <feature bit="0" code="SMOKE" name="SmokeAlarm" summary="Supports Smoke alarm">
-        <optionalConform choice="a" more="true"/>
+        <optionalConform choice="a" more="true" min="1"/>
       </feature>
-      <feature bit="1" code="CO" name="CO Alarm" summary="Supports CO alarm">
-        <optionalConform choice="a" more="true"/>
+      <feature bit="1" code="CO" name="COAlarm" summary="Supports CO alarm">
```

Comment:
Looks like this change in the feature name is a source-compat-breaking change in the C++ API, because that API has silly naming conventions?